individual/models.py

```diff
@@ -13,12 +13,17 @@ class Meta:
         managed = True
 
 
-class IndividualDataSource(HistoryModel):
+class IndividualDataSourceUpload(HistoryModel):
```

Comment:
Please add fields 
Status - ENUM 
Error - JSON
to match specification from ticket

Reply:
Done.